packages/node_modules/@node-red/runtime/lib/flows/Flow.js

```diff
@@ -818,6 +818,17 @@ function handlePreRoute(flow, sendEvent, reportError) {
     })
 }
 
+function deliverMessageToDestination(sendEvent) {
+    if (sendEvent.destination.node) {
```

Comment:
```suggestion
    if (sendEvent?.destination?.node) {
```

I think we might want to be a bit more guarded here in case `sendEvent` or `destination` are null/undefined

alternatively, wrap whole thing in try/catch